### PR TITLE
New script: Mylar3 LXC

### PR DESCRIPTION
**Pull request by davalanche**
> [!NOTE]
> We are meticulous when it comes to merging code into the main branch, so please understand that we may reject pull requests that do not meet the project's standards. It's never personal. Also, game-related scripts have a lower chance of being merged.

## Description

I have created a new LXC: Mylar3.

## Type of change
Please check the relevant option(s):

- [ ] Bug fix (non-breaking change that resolves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [X] New script (a fully functional and thoroughly tested script or set of scripts.)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [X] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [X] Testing performed (I have tested my changes, ensuring everything works as expected)
- [X] Documentation updated (I have updated any relevant documentation)

## Additional Information (optional)
 - I chose to go with a git source install (clone) vs. a git release as the former allows for application updates through the user interface.
 - The unrar install logic was copied from the SABnzbd script.

## Related Pull Requests / Discussions

- Related Discussion #312

### CODE REVIEW

File: install/mylar3-install.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  python3-pip
+echo "deb http://deb.debian.org/debian bookworm non-free non-free-firmware" > /etc/apt/sources.list.d/non-free.list
+$STD apt-get update
+$STD apt-get install -y unrar
+rm /etc/apt/sources.list.d/non-free.list
+msg_ok "Installed Dependencies"
+
+msg_info "Installing ${APPLICATION}"
+mkdir -p /opt/mylar3 && mkdir -p /opt/mylar3-data
+$STD git clone -b master https://github.com/mylar3/mylar3.git /opt/mylar3
```
[the remark this answers is not in the document]
Comment (davalanche) [reply install/mylar3-install.sh:32]: I started with a release based install but read in another discussion or PR review that if the installation method allows the application to update itself that is preferred?  If that is not the case though I will switch back to using the releases.  Thanks!

[the remark this answers is not in the document]
Comment (havardthom) [reply install/mylar3-install.sh:32]: Release based install is preferred over git clone. But you don't need to add update function if you can update the app through web interface

File: install/mylar3-install.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED
+$STD pip install -U --no-cache-dir pip
```
[the remark this answers is not in the document]
Comment (davalanche) [reply install/mylar3-install.sh:34]: I will test this but the idea to upgrade pip first and then do the requirements install next came from this [Dockerfile](https://github.com/linuxserver/docker-mylar3/blob/master/Dockerfile) (lines 43+46).  pip is not a requirement of the application so I don't know if the command you suggest will see it updated - I will report back.

[the remark this answers is not in the document]
Comment (davalanche) [reply install/mylar3-install.sh:34]: I did some testing and can confirm that the single command will not see pip updated.

File: install/mylar3-install.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Type=forking
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+EOF
+systemctl enable -q --now mylar3.service
+msg_ok "Created Service"
+
+msg_info "Updating ${APPLICATION} configuration"
```
[the remark this answers is not in the document]
Comment (davalanche) [reply install/mylar3-install.sh]: Removed the configuration changes entirely as they were only required when installing via git clone.

**Review by MickLesk — APPROVED** [review]

**Pull request merged**